### DISCUSSION TITLE
[RISOLTO] Errore di caricamento Supervisor store

### DISCUSSION THREAD

**Discussion by federicobravin03** [indomus/forum#658] [HUB: Home Assistant]
Ciao a tutti, ho appena installato HA, però nella store in cui si possono scaricare le integrazioni, non me ne compare nessuna. Andando a vedere il log del Supervisor compaiono questa serie di errori (allego screen). Spero che qualcuno di voi mi possa aiutare. grazie in anticipo.

![image](https://user-images.githubusercontent.com/74401403/131220550-1de88cc8-9d7e-4665-9d71-cf1abd083c70.png)

**Comment by snakuzzo** [comment]
Problemi di rete / DNS

**Reply by federicobravin03** [reply]
Grazie, ho risolto ripristinando la configurazione del router.
